spec/cypress/integration/Cookies.feature

```diff
@@ -0,0 +1,40 @@
+Feature: Cookie page
+  Everybody wants to set cookie preferences
```

Comment:
And rule the world, but they rarely get to

Reply:
Yeah - could we change the scenario text for that. I'm sure we can find a few exceptions to "Everybody" - my nice for example is more interested in painting all the walls in all sorts of colors and cookie preferences are not on her agenda for the next 7 years

Reply:
This is just example text, slightly changed from the file I copy and pasted from the cypress-cucumber docs 😆

Reply:
;) Let's make it less inclusive and less accessible